ci-operator/step-registry/redhat-chaos/pod-scenarios/random-system-pods/redhat-chaos-pod-scenarios-random-system-pods-ref.yaml

```diff
@@ -0,0 +1,26 @@
+ref:
+  as: redhat-chaos-pod-scenarios-random-system-pods 
+  from_image:
+    namespace: chaos
+    name: krkn
+    tag: latest
+  commands: redhat-chaos-pod-scenarios-random-system-pods-commands.sh
+  env:
+  - name: TARGET_NAMESPACE
+    default: "openshift-*"
+  - name: POD_LABEL
```

Comment:
this looks to still be being set in the regex scenario

Reply:
https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_release/37359/rehearse-37359-periodic-ci-redhat-chaos-krkn-hub-main-krkn-hub-tests/1638653857963708416/artifacts/krkn-hub-tests/redhat-chaos-pod-scenarios-random-system-pods/build-log.txt